### PR TITLE
[stable/rabbitmq-ha] Added 'topicPermission' key in definitions.json secret

### PR DESCRIPTION
**Pull request by andreagilardoni**
Added 'topicPermission' key in definitions.json secret because it was missing

### CODE REVIEW

**Review by steven-sheehy — CHANGES_REQUESTED** [review]
This needs a minor version increase to the Chart.yaml and please sign the DCO. Other than that it looks good

**Comment by alcheagle** [comment]
Is there something else that I need to do in order to complete this pull request?

**Comment by steven-sheehy** [comment]
> Is there something else that I need to do in order to complete this pull request?

See my previous statement

**Comment by alcheagle** [comment]
Sorry, I forgot to change email in my commits, now it seems to be ok.

**Comment by steven-sheehy** [comment]
And yet you still have not bumped the chart version as asked. Please ensure all GitHub checks pass.

File: stable/rabbitmq-ha/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: rabbitmq-ha
 apiVersion: v1
 appVersion: 3.8.0
-version: 1.40.1
+version: 1.40.2
```
Comment (steven-sheehy) [inline stable/rabbitmq-ha/Chart.yaml]: Once again, this needs a minor version increase for new features. Please see semver.org

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

File: stable/rabbitmq-ha/Chart.yaml (hunk 1 above)
Comment (andreagilardoni) [reply stable/rabbitmq-ha/Chart.yaml]: Then what do I have to put? 1.41.0, correct?

Comment (steven-sheehy) [reply stable/rabbitmq-ha/Chart.yaml]: 1.41.0 currently, yes. You also now need to fix the merge conflict.

**Review by steven-sheehy — APPROVED** [review]
/lgtm
Thanks

**Pull request merged**